web/template/home.gohtml

```diff
@@ -567,7 +567,9 @@
                         </section>
                     </template>
                     <template x-if="plannedStreams.hasElements()">
-                        <section class="tum-live-course-view-item px-3">
+                        <section x-data="{isLectureHallValid(lectureHall) {const regex = /^\d{4}\.[A-Z0-9]{2}\.[A-Z0-9]{3,4}$/;
```

Comment:
Please move this function into the corresponding typescript file. Especially functions shouldn't be in the x-data property as this should only contain data.